### PR TITLE
cloud: update quota-based invoice doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update quota-based invoice doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-billing.md [hunk 1]
```diff
@@ -15,11 +15,17 @@ TiDB Cloud charges according to the resources that you consume. You can visit [T
...
 
+After you set up the payment method, TiDB Cloud will generate an invoice once your cost reaches a quota, which is $500 by default. If you want to raise the quota or receive one invoice per month, you can [contact our sales](https://www.pingcap.com/contact-us/).
+
 > **Note:**
 >
-> If you sign up for TiDB Cloud through [AWS Marketplace](https://aws.amazon.com/marketplace), you can pay through your AWS account directly but cannot add payment methods or download invoices in the TiDB Cloud portal.
+> If you sign up for TiDB Cloud through [AWS Marketplace](https://aws.amazon.com/marketplace) or [Google Cloud Marketplace](https://cloud.google.com/marketplace), you can pay through your AWS account or Google Cloud account directly but cannot add payment methods or download invoices in the TiDB Cloud portal.
+
+After you contact our sales for receiving an invoice on a monthly basis or if you signed up for TiDB Cloud through AWS Marketplace or Google Cloud Marketplace, TiDB Cloud will generate the invoice for the previous month at the beginning of each month.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-billing.md:24]:
```suggestion
After you contact our sales for receiving an invoice on a monthly basis, TiDB Cloud will generate the invoice for the previous month at the beginning of each month.
```

**Comment by nizheming** [comment]
LGTM

File: tidb-cloud/tidb-cloud-billing.md [hunk 2]
```diff
@@ -15,11 +15,17 @@ TiDB Cloud charges according to the resources that you consume. You can visit [T
...
+> If you sign up for TiDB Cloud through [AWS Marketplace](https://aws.amazon.com/marketplace) or [Google Cloud Marketplace](https://cloud.google.com/marketplace), you can pay through your AWS account or Google Cloud account directly but cannot add payment methods or download invoices in the TiDB Cloud portal.
+
+After you contact our sales for receiving an invoice on a monthly basis, TiDB Cloud will generate the invoice for the previous month at the beginning of each month.
+
+Invoice costs include TiDB cluster usage consumption, discounts, backup storage costs, support service cost, credit consumption, and data transmission costs in your organization.
 
-After you set up the payment method, TiDB Cloud will generate the invoice for the previous month at the beginning of each month. Invoice costs include TiDB cluster usage consumption, discounts, backup storage costs, support service cost, credit consumption, and data transmission costs in your organization.
+For each monthly invoice:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:28]: each monthly invoice or each invoice?

File: tidb-cloud/tidb-cloud-billing.md [hunk 3]
```diff
@@ -15,11 +15,17 @@ TiDB Cloud charges according to the resources that you consume. You can visit [T
...
 
 - TiDB Cloud provides the invoice to you on the ninth of each month. From the first to the ninth day, you cannot view the last month's cost details, but can obtain the cluster usage information of this month via the billing console.
 - The default method for paying invoices is credit card deduction. If you want to use other payment methods, please send a ticket request to let us know.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:31]: Are L31 to L32 also applicable to non-monthly invoices?

Comment (TomShawn) [reply tidb-cloud/tidb-cloud-billing.md:31]: As confirmed, updated in https://github.com/pingcap/docs/pull/10470/commits/b3bf5d9c6ade4b94889e364c9828d9db8ec89cdc

**Review by qiancai — APPROVED** [review]

**Pull request merged**